go.mod

```diff
@@ -1,6 +1,6 @@
 module get.porter.sh/porter
 
-go 1.17
+go 1.18
```

Comment:
I see that this caused the go.sum file to change, but at least it doesn't impact the go.mod file like the upgrade to 1.17 did. We may run into the sum file changing for no reason depending on which version of go a developer is running but hopefully that doesn't cause errors like when you add a go mod entry and forget to run tidy. We'll see I guess. 😁